src/DescriptorSet.cc

```diff
@@ -37,12 +37,13 @@ DescriptorSet::~DescriptorSet()
 bool
 DescriptorSet::add(int fd)
 {
-    assert(0 <= fd && fd < capacity_); // \todo: replace with Must()
+    Must(0 <= fd);
+    Must(fd < capacity_);
```

Comment:
@yadij, would you mind removing this and similar `s/assert/Must/` changes from this PR, so that we do not have to argue about them and their details? Combining comment changes with any "actual code" changes makes the scope of this PR a lot more complex and complicates the PR review quite a bit!

Reply:
Done in 117408db0d48d25c18ad97034e6b666cd7cf33f9

Reply:
NP: the other notable logic change is mem/Pool.h and mem/Pool.cc everything else is documentation.